Source/UI/NewUI/PaymentForm/Default Implementation/Cardholder/DefaultCardholderFormStyle.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  DefaultCardholderFormStyle.swift
+//  
+//
+//  Created by Alex Ioja-Yang on 16/08/2022.
+//
+
+import UIKit
+
+public struct DefaultCardholderFormStyle: CellTextFieldStyle {
```

Comment:
Do we need to have Form in `DefaultCardholderFormStyle` ? Can this just be `DefaultCardholderStyle` ?

Reply:
I have kept the formatting with similar components "DefaultCardNumberFormStyle", "DefaultExpiryDateFormStyle" & "DefaultPayButtonFormStyle". I don't mind dropping the Form keyword but we'd need to do it across all the elements so we follow some pattern.